Building with module "random" requires building with "xtimer" also to succeed.

Building with just "random" results in the following error:

```sys/shell/commands/sc_random.c: In function '_random_init':
sys/shell/commands/sc_random.c:51:5: error: 'initval' may be used uninitialized in this function [-Werror=maybe-uninitialized]
     genrand_init(initval);```

--- a/sys/shell/commands/sc_random.c
+++ b/sys/shell/commands/sc_random.c
@@ -40,7 +40,9 @@ int _random_init(int argc, char **argv)
         printf("PRNG initialized to current time: %d\n", initval);
 #else
         (void)initval;
-        printf("xtimer module not compiled in, can't initialize by time.\n");
+        puts("xtimer module not compiled in, can't initialize by time.\n"
+             "Please provide a seed.\n");
+        return 1;
 #endif
     }
     else {

